force-app/main/default/classes/ProgramEngagementSelector_TEST.cls

```diff
@@ -124,9 +124,9 @@ public with sharing class ProgramEngagementSelector_TEST {
                 SELECT Id, Contact__r.Name, Contact__r.Email, Stage__c, ProgramCohort__c
                 FROM ProgramEngagement__c
                 WHERE
-                    Stage__c IN :ACTIVE_ENGAGEMENT_STAGES
-                    AND Program__c = :program.Id
+                    Program__c = :program.Id
                     AND Contact__r.Name LIKE '%Contact%'
+                    AND Stage__c = 'Enrolled'
```

Comment:
I set the expected result specifically to 'Enrolled' vs what it was before with just the active stages.  This is because I also pass in the 'Enrolled' value to the method for the actual result.  Happy to change if you think something else should be happening here.